cli/util.go

```diff
@@ -896,3 +896,11 @@ func calculateRate(new, last float64, since time.Duration) float64 {
 
 	return (new - last) / since.Seconds()
 }
+
+func parseApiLevel(lvl string) uint {
```

Comment:
move to internal/util - sorry should have been clearer since there are multiple utils, trying to minimise noise in the cli package